src/ganache-server.ts

```diff
@@ -9,25 +9,19 @@ function send(msg: Message): void {
   process.send(msg);
 }
 
-function setupServer({ accountsConfig, gasLimit, gasPrice, coverage }: Options) {
-  const ganacheOpts = {
-    accounts: accountsConfig,
-    gasLimit,
-    gasPrice: `0x${gasPrice.toString(16)}`,
-  };
-
-  if (!coverage) {
-    return ganache.server(ganacheOpts);
+function setupServer(nodeOptions: NodeOptions): any {
+  if (!nodeOptions.coverage) {
+    return ganache.server(nodeOptions);
   } else {
     return require('ganache-core-coverage').server({
-      ...ganacheOpts,
+      ...nodeOptions,
       emitFreeLogs: true,
       allowUnlimitedContractSize: true,
```

Comment:
Note that this may collide with the same field inside `nodeOptions`. Why are we adding it to the configuration? I didn't know there were use cases for it outside of coverage.

Reply:
Use cases of what `allowUnlimitedContractSize`? We are adding it because [it was requested by community](https://github.com/OpenZeppelin/openzeppelin-test-environment/pull/55) and it certainly has use cases outside test coverage.
Because it may collide with `nodeOptions`,  `allowUnlimitedContractSize` comes after to make sure it is always `true`.
And finally since the next PR is solidity-coverage this code doesn't have much of timespan left.